.github/workflows/ci_tests.yaml

```diff
@@ -73,7 +73,7 @@ jobs:
             numpy-version: '1.25'
             pandas-version: '=2.0'
             xarray-version: '=2023.04'
-            optional-packages: ' contextily geopandas<1 ipython pyarrow rioxarray sphinx-gallery'
+            optional-packages: ' contextily geopandas<1 ipython pyarrow-core rioxarray sphinx-gallery'
```

Comment:
Also need to update line 82.

Do we also want to update `doc/install.md`?

Reply:
Good catch, done in commits 3e59046581ba796912b0a475391d17bdce6d9d82 and 0cef357ae4a76e95456c7166d442d59ec5c48e3b